docs/test-pages/TCD-002.adoc

```diff
@@ -0,0 +1,33 @@
+= TCD-002
+
+== Purpose
+
+Checking that the IDE knows the jsl file extension as a JSL file.
```

Comment:
I prefer 'Checking that the IDE recognizes a file with .jsl extension as a JSL file.' here